src/ng/compile.js

```diff
@@ -1047,6 +1047,9 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
    *      See {@link ng.$compile#-bindtocontroller- `bindToController`}.
    *    - `transclude` – `{boolean=}` – whether {@link $compile#transclusion content transclusion} is enabled.
    *      Disabled by default.
+   *    - `require` - `{!Object|undefined}` - requires the controllers of other directives and binds them to
```

Comment:
Why `!Object|undefined` instead of `object=` (as other parameters above) ?
Besides, it can be a string or an array as well.